dtfs-central-api/src/v1/controllers/tfm/deal/tfm-submit-deal.controller.js

```diff
@@ -90,11 +90,55 @@ const createFacilitiesSnapshot = async (deal) => {
   return { status: 400, message: 'Invalid Deal Id' };
 };
 
+/**
+ * creates amendments object in tfm-facilities
+ * object blank history array
+ */
+const createFacilitiesAmendments = async (deal) => {
```

Comment:
shouldn't this be on `facilities` side?

Reply:
needs to be here as is part of the submit deal (which also creates the facilitySnapshot here)

Reply:
removed